src/Sylius/Behat/Page/Shop/Product/ShowPage.php

```diff
@@ -318,6 +326,7 @@ protected function getDefinedElements()
             'option_select' => '#sylius_add_to_cart_cartItem_variant_%option-name%',
             'out_of_stock' => '#sylius-product-out-of-stock',
             'product_price' => '#product-price',
+            'product_original_price' => '#product-original-price',
```

Comment:
Wrong order in the array

Reply:
Why order is wrong , what is the correct order ?

Reply:
`product_original_price` key should be before `product_price` key :)

Reply:
Alphabetical order is life 🔤